test/stress.test.js

```diff
@@ -387,10 +397,10 @@ describe('#integration stress tests', () => {
     }
   }
 
-  function verifyQueryResult (result) {
+  function verifyQueryResult (result, context) {
     if (!result) {
       return new Error('Received undefined result')
-    } else if (result.records.length === 0) {
+    } else if (result.records.length === 0 && context.writeCommandsRun < 20) {
```

Comment:
Why 20?

Reply:
It was totally arbitrary. I tried to put whenever we had run any write commands, but it doesn't work because of the parallelism and we don't know if the write command finished before the read starts. And since we have 24 executors, 20 seems to be a reasonable number to start to check if the length is not zero. 

I will put this as function of the number of executors.